xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -926,11 +926,13 @@ void CRenderManager::UpdateDisplayLatency()
   float refresh = fps;
   if (g_graphicsContext.GetVideoResolution() == RES_WINDOW)
     refresh = 0; // No idea about refresh rate when windowed, just get the default latency
-  m_displayLatency = (double) g_advancedSettings.GetDisplayLatency(refresh);
-
-  int buffers = g_Windowing.NoOfBuffers();
-  m_displayLatency += (buffers - 1) / fps;
+  m_displayLatency = g_advancedSettings.GetDisplayLatency(refresh);
```

Comment:
The latency overall was totally static before which is no longer possible. If we reuse m_displayLatency for the dynamically updated latency, we would have to call `CAdvancedSettings::GetDisplayLatency` on every frame - at the moment it is only called when resolution changes. But I agree that it would be better to show the actually used value in the UI. So what do you think: Call `CAdvancedSettings::GetDisplayLatency` every frame anyway or introduce an extra variable for the dynamic part?